charts/sam/templates/ingress.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
IMO it might be worth setting a TLS policy on the Ingress now rather than later, otherwise it'll probably just show up in prod scans again later and require remediation:

https://broadworkbench.atlassian.net/browse/DDO-494
https://broadworkbench.atlassian.net/browse/DDO-480